functionary/train/custom_datasets.py

```diff
@@ -377,6 +446,42 @@ def merge_data_points(data_points: List[Dict], tokenizer: Any) -> Dict:
     }
 
 
+def pack_data_points_FA(data_points: List[Dict], tokenizer: Any) -> Dict:
```

Comment:
Can you provide docstring and more comments before I review this?

Reply:
sure